README.md

```diff
@@ -134,27 +135,27 @@ You should mount `/var/log/modsecurity` onto a persistent volume.
 Configured directories and files are:
 
 * MODSEC_AUDIT_LOG (Default: `/var/log/modsecurity/audit.log`)
-  - `SecAuditLog`
+  * `SecAuditLog`
 * MODSEC_AUDIT_STORAGE (Default: `/var/log/modsecurity/audit`)
-  - `SecAuditLogStorageDir`
+  * `SecAuditLogStorageDir`
 
 #### Ephemeral
 
 Following the 12-factor app guidelines we're logging error and access
 logs to the console, and let the cluster's ELK stack deal with logging:
 
 * ErrorLog (Default: `/dev/stderr`)
-  - *see* APACHE_ERRORLOG
+  * *see* APACHE_ERRORLOG
 * CustomLog (Default: `/dev/stdout`)
-  - *see* APACHE_ACCESSLOG
-  - *see* APACHE_PERFLOG
+  * *see* APACHE_ACCESSLOG
+  * *see* APACHE_PERFLOG
 
 You should mount `/tmp/modsecurity` onto a scratch space, such as an
 `emptyDir` volume. Configured settings are:
 
 * MODSEC_DATA_DIR (Default: `/tmp/modsecurity`)
-  - `SecDataDir`
+  * `SecDataDir`
 * MODSEC_TMP_DIR (Default: `/tmp/modsecurity`)
-  - `SecTmpDir`
+  * `SecTmpDir`
 * MODSEC_UPLOAD_DIR (Default: `/tmp/modsecurity`)
```

Comment:
Since there is a separate folder configured this should be reflected in the README (`/tmp/modsecurity-upload`)